src/linse/annotate.py

```diff
@@ -533,7 +533,8 @@ def seallable(
 
     if 3 <= len(sequence) <= 5:
         # first element must be the initial
-        ini = 'i' if cv[0] == 'C' else '?'
+        #ini = 'i' if cv[0] == 'C' else '?'
+        ini = 'i' if cv[0] == 'C' or cv[0]== 'V' else '?'
```

Comment:
@macyl, if cv[0] is "V", there can't be an initial, the initial is nothing in this case.

Reply:
ok, I understand. So as long as a vowel is at the start of the string, the return string is "nct". I will change it now :) (sorry, I took today as a reading day + class day).